Use the default option shift arrow key selection for the TextMate keymap

jish opened the pull request:
By default when pressing the option+shift+arrow keys selection should jump to words along the way rather than selecting the entire line.

https://github.com/zed-industries/zed/issues/10242


Release Notes:

- Fixed ([#10242](https://github.com/zed-industries/zed/issues/10242)).

TextMate keymap uses default option shift arrow selection

jish committed:
TextMate keymap uses default option shift arrow selection

By default when pressing the option+shift+arrow keys selection should
jump to words along the way rather than selecting the entire line.

https://github.com/zed-industries/zed/issues/10242

mrnugget commented on assets/keymaps/textmate.json:
```diff
@@ -39,13 +39,13 @@
       "alt-shift-left": [
-        "editor::SelectToBeginningOfLine",
+        "editor::SelectToPreviousSubwordStart",
         {
           "stop_at_soft_wraps": true
         }
       ],
       "alt-shift-right": [
-        "editor::SelectToEndOfLine",
+        "editor::SelectToNextSubwordEnd",
```
Are you sure you want `Subword` here instead of `Word`?

jish replied:
@mrnugget yes, that makes sense.

Updated here: https://github.com/zed-industries/zed/pull/10535/commits/c452d2a3dc4172622f7dcfb807bee2cf37ab4668

jish committed:
TextMate alt-shift-arrow selects whole words

This was just a typo in the previous commit. It was meant to be
`SelectToPreviousWordStart` and `SelectToNextWordEnd` in the first
place.

mrnugget commented:
Closing this since it's been superseded by https://github.com/zed-industries/zed/pull/11292
